### PR TITLE
FAB-144: Create Vue 3 SDK for Fabra Connect

### PR DESCRIPTION
**Pull request by yuliswe**
Create vue-fabra-connect component module for Vue 3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,12 +25,11 @@ export default {
...
-  <section>
-    <fabra-connect :linkToken="ADD_GENERATED_LINK_TOKEN" /* :customTheme="customTheme" */>
-      <template slot="button" slot-scope="props">
-        <button @click="props.onClick">Open Fabra Connect</button>
-      </template>
-    </fabra-connect>
-  </section>
+  <div id="fabra-connect-container"></div>
+  <fabra-connect :linkToken="ADD_GENERATED_LINK_TOKEN" containerID="fabra-connect-container">
+    <template #button="props">
```
Comment (yuliswe) [inline README.md:30]: `#button="props"` is the new syntax. `slot="button" slot-scope="props"` is from Vue2 and is deprecated in Vue3. I'm not sure where the merge-link library got this but I tried the old syntax and it doesn't work. :/ Here's the reference I found: 

https://stackoverflow.com/questions/57806610/how-to-use-slot-scope-correctly-within-vuejs-2-5-x
Old syntax:
https://v2.vuejs.org/v2/guide/components-slots.html#Scoped-Slots-with-the-slot-scope-Attribute

New syntax:
https://vuejs.org/guide/components/slots.html#scoped-slots

File: package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "dist"
+  ],
+  "main": "./dist/fabra-connect.umd.js",
+  "module": "./dist/fabra-connect.es.js",
+  "types": "./dist/main.d.ts",
+  "scripts": {
+    "build": "vue-tsc && vite build"
+  },
+  "peerDependencies": {
+    "vue": "^3.2.45"
```
Comment (yuliswe) [inline package.json]: Please verify this version is ok

File: src/fabra-connect.vue [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        document.head.appendChild(el);
+      });
+    },
+  },
+  watch: {
+    $props: {
+      deep: true,
+      handler() {
+        this.destroy()
+        this.initialize();
```
Comment (yuliswe) [inline src/fabra-connect.vue]: Is this ok? I looked at the `initialize.js` code to find a way to `updateTheme`. There's a `updateTheme` function in the main branch but that code isn't released yet.

**Review by yuliswe — COMMENTED** [review]
As before, I created an example Vue app that uses this library:

https://github.com/yuliswe/vue-fabra-connect-example

File: src/fabra-connect.vue (hunk 3 above)
Comment (yuliswe) [reply src/fabra-connect.vue]: Without that the only way to reload the theme is to reload the instance.

File: package.json (hunk 2 above)
Comment (nfiacco) [reply package.json]: can we just do: `"^3.0.0"`

File: src/fabra-connect.vue (hunk 3 above)
Comment (nfiacco) [reply src/fabra-connect.vue]: I don't think we need a way to update the theme— do you mean we need `updateTheme` just to show the theme on reloads?

File: package.json (hunk 2 above)
Comment (yuliswe) [reply package.json]: done

**Review by nfiacco — APPROVED** [review]

**Pull request merged**